src/ParallelAlgorithms/Interpolation/Targets/Sphere.cpp

```diff
@@ -0,0 +1,32 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "ParallelAlgorithms/Interpolation/Targets/Sphere.hpp"
+
+#include <array>
+#include <cstddef>
+#include <pup.h>
+
+#include "Options/Options.hpp"
+
+namespace intrp::OptionHolders {
+Sphere::Sphere(size_t l_max_in, std::array<double, 3> center_in,
```

Comment:
you can `const` the arguments here. Only asking because the dox comment needs to be moved anyway